src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/cost/McCostParams.java

```diff
@@ -75,6 +80,10 @@ public WheelchairAccessibilityRequest accessibilityRequirements() {
     return accessibilityRequest;
   }
 
+  public Map<FeedScopedId, Integer> routePenaltyMap() {
```

Comment:
I think the word "map" in the method name is superfluous. I would just call the method `routePenalties`.